extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -88,12 +91,17 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 	try {
 		const result = await azureFunctionsService.addSqlBinding(selectedBinding.type, uri.fsPath, azureFunctionName, objectName, connectionStringSetting);
 
+		TelemetryReporter.createActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.addSqlBinding)
```

Comment:
You should only be sending this if the result is a success, right? And sending an error event if it's not?